tools/acrntrace/README.rst

```diff
@@ -18,8 +18,11 @@ output to trace file under ``/tmp/acrntrace`` with raw (binary) data format.
 Options:
 
 -h			print this message
--t period		specify polling interval in milliseconds [1-999]
+-i period		specify polling interval in milliseconds [1-999]
+-t			max time to capture trace data (in second)
```

Comment:
If the -t option has a parameter you should mention that:

    -t time            max time to capture trace data (in seconds)